controllers/contents_controller.go

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import (
+	"../modules/logging"
+	"github.com/flosch/pongo2"
+	"github.com/zenazn/goji/web"
+	"net/http"
+)
+
+type Contents struct {
```

Comment:
__[golint]__
* [golint] exported type Contents should have comment or be unexported